modules/fedimint-ln/tests/ln_contracts.rs

```diff
@@ -22,8 +23,8 @@ use secp256k1::KeyPair;
 
 fn ln_decoders() -> ModuleDecoderRegistry {
     ModuleDecoderRegistry::new([(
-        MODULE_KEY_LN,
-        Decoder::from_typed(&fedimint_ln::common::LightningModuleDecoder),
+        LEGACY_HARDCODED_INSTANCE_ID_LN,
```

Comment:
I know its out of scope for this PR, but what is the plan for this value? Will this be generated when the module is registered?

Reply:
The plan is not to have it, and tests and client do a bit of an extra effort not to assume that that a given module has a fixed instance id.

We will have to do this work one way or another, if we want to have client code that e.g. can work with a Federation without a mint, or LN, or wallet, etc.

But client code is largely not modularized and in a lot of places ... shotcuts have been taken. :D

Reply:
I think one aspect of this PR is that it kind of guides the developer into thinking about module's optionality. Even if we don't see immediate use for multiple modules, we will want support some modules not being there at least.